requirements.txt

```diff
@@ -9,3 +9,4 @@ pandas>=1.0.1
 numba>=0.52.0
 matplotlib>=3.3.3
 pyyaml
+tensorflow
```

Comment:
The tensorflow requirement can be hefty for some users, especially if they aren't intending to use the analysis functions of paltas. There is currently a note in the readme about installing tensorflow if you want to use the analysis functions. It would be best to keep this out. We may want to be more explicit about separating the analysis features moving forward.

Reply:
OK! Removed this line.